config/checkstyle_checks.xml

```diff
@@ -219,7 +219,8 @@
   <module name="LineLength">
     <property name="fileExtensions" value="java"/>
     <property name="max" value="100"/>
-    <property name="ignorePattern" value="^ *\* *([^ ]+|\{@code .*|&lt;a href=&quot;[^&quot;]+&quot;&gt;)$"/>
+    <property name="ignorePattern"
+              value="^( *\* *([^ ]+|\{@code .*|&lt;a href=&quot;[^&quot;]+&quot;&gt;)|(package|import) .*)$"/>
```

Comment:
```
           import a.b.c; // wont match
import a.b.c; // would match
```
I deliberately did not modify the regex to match the former one as for our project (or any project) starting import from the start of the line is the best practice.